spec/fluent.ebnf

```diff
@@ -31,8 +32,7 @@ word                 ::= (((char - line-break) - inline-space) - [#x5b#x5c#x5d#x
 builtin              ::= [A-Z] [A-Z_?-]*
 number               ::= '-'? [0-9]+ ('.' [0-9]+)?
 
-variant-key          ::= number | variant-symbol
-variant-symbol       ::= word (_ word)*
+variant-key          ::= word (_ word)*
```

Comment:
Is this change intentional?

Reply:
Yes. The `word` production already allows all characters that `number` did.

Reply:
Hmm, but having two numbers next to each other separated by a space is not what we want. OK, I'll bring this back.